okhttp/src/main/java/okhttp3/RealCall.kt

```diff
@@ -115,7 +115,7 @@ internal class RealCall private constructor(
       assert(!Thread.holdsLock(client.dispatcher()))
       var success = false
       try {
-        executorService.execute(this)
+        executorService.execute("OkHttp ${redactedUrl()}", this@AsyncCall::run)
```

Comment:
This is breaking the test expectations of RecordingExecutor :(

Reply:
I was trying to be helpful but maybe overstepped a line on the PR.  No issue if you revert :)

Reply:
yeah, I fixed